mailpoet/assets/js/src/automation/integrations/woocommerce-subscriptions/steps/subscription-status-changed/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { __, _x } from '@wordpress/i18n';
+import { StepType } from '../../../../editor/store';
+import { Icon } from './icon';
+import { PremiumModalForStepEdit } from '../../../../../common/premium-modal';
+
+const keywords = [
+  __('woocommerce', 'mailpoet'),
+  __('subscriptions', 'mailpoet'),
+  __('status', 'mailpoet'),
+  __('change', 'mailpoet'),
+];
+
+export const step: StepType = {
+  key: 'woocommerce-subscriptions:subscription-status-changed',
+  group: 'triggers',
+  title: () => __('Woo subscription status changed', 'mailpoet'),
+  description: () =>
+    __(
+      'Start the automation when subscription changed to a specific status.',
+      'mailpoet',
+    ),
+  subtitle: () => _x('Trigger', 'noun', 'mailpoet'),
+  keywords,
+  foreground: '#2271b1',
+  background: '#f0f6fc',
+  icon: () => <Icon />,
+  edit: () => (
+    <PremiumModalForStepEdit
+      tracking={{
+        utm_medium: 'upsell_modal',
+        utm_campaign: 'create_automation_editor_subscription_status_changed',
+      }}
+    >
+      {__(
+        'Starting an automation by changing the status of an subscription is a premium feature.',
```

Comment:
If kept as-is this should be `a subscription` instead of `an`. I don't think the `by` phrasing sounds as natural as it could. In my opinion it sounds better as `Starting an automation when a WooCommerce subscription changes status is a premium feature.`

Reply:
Thanks a lot for those improvements! 35eeb78f2